app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Please refer to [Template guide](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-overview.html) and [HTML code standard](https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-html.html). Fix all templates

Reply:
fixed